src/engines/aurora/gui.cpp

```diff
@@ -70,6 +70,11 @@ void GUI::show() {
 			widget.show();
 	}
 
+	std::list<GUI*>::iterator iter;
+	for (iter = _childrenguis.begin(); iter != _childrenguis.end(); iter++)	{
```

Comment:
I guess you defined the iterator before the for-loop because of the line length? In general, I prefer to have the definition within the loop, to limit the scope of the variable